src/main/scala/tech/cryptonomic/conseil/tezos/michelson/renderer/MichelsonRenderer.scala

```diff
@@ -9,20 +9,27 @@ object MichelsonRenderer {
     def render(): String = self match {
 
       // instructions
-      case MichelsonSingleInstruction(prim, List(), List()) => prim
-      case MichelsonSingleInstruction(prim, args, annotations) => s"$prim ${(annotations ++ args.map(_.render())).mkString(" ")}"
+      case MichelsonSingleInstruction(name, List(sequence1: MichelsonInstructionSequence, sequence2: MichelsonInstructionSequence), _) => {
+        val indent = " " * (name.length + 1)
+        val embeddedIndent = " " * (name.length + 3)
```

Comment:
`indent + " " * 2` is more clear? or I might have misinterpreted intent

If the two concept are tied, why not the definitions? 😉

Reply:
What about multiple nesting? Does this handle the case?

Reply:
I tried with ``def`` but it looked worse for me. It supports multiple nesting. I added a test for that case.

Reply:
I didn't meant to use `def`, just to reuse the `ident` val inside the definition of `embeddedIdent`